modules/ROOT/pages/mule-app-properties-to-configure.adoc

```diff
@@ -23,9 +23,14 @@ http.port=10000
 http.host=my-api.cloudhub.io
 ----
 
+MuleSoft recommends using `.yaml` files because it supports the addition of type validations and autocompletion.
+
 == Create and Configure your Properties File
 
-. Create a new `.yaml` or `.properties` file in your project, and store it in `/src/main/resources`.
+. In Studio, go to the *Package Explorer* view and right click the `/src/main/resources` folder in your project. Then, select *New* > *File*:
```

Comment:
This is a lot of user activity for just one step. How about...:
```suggestion
. In Studio, go to the *Package Explorer* view and right-click the `/src/main/resources` folder in your project. 
. Select *New* > *File*:
```
Note also hyphenation of "right-click."